src/classes/TaskDetailController.cls

```diff
@@ -0,0 +1,30 @@
+public with sharing class TaskDetailController {
+	@AuraEnabled
+	public static boolean changeStatus(HTask__c task, string status) {
+        
+		if (status == 'Completed'){
```

Comment:
Why are these if statements here if the only thing you do is set the same string as the task's status? You can simply do a sinle task.Status__c = status